project.html

```diff
@@ -1,63 +1,69 @@
-              would mostly show information and images and it would be a testing
-              ground of me using what i learned about html and css at a begginer
-              level.This includes adding images changing text size, adding background colure,
-              and adding a table.
-            </p>
-          </div>
-        </div>
-        <div class="flex">
-          <div class="box">
-            <img src="Appcard.jpg" height="400" width="600" />
-            <h2>Yugioh wiki moblie app</h2>
-            <p>
-              for this project is mosly a app about a certin type of game
-              showing information and images. this was mostly a 3 or more month
-              project coding the app for the first time using a sotfware tool of
-              andriod studio and using the language called kotlin.
-            </p>
-          </div>
-        </div>
-      </div>
-    </div>
-
-    <footer class="foot">
-      <p>
-        by chidera njoku <br />
-        created:2024
-      </p>
-    </footer>
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About me</title>
+    <link rel="stylesheet" href="aboutme.css" />
+  </head>
+  <body>
+    <div>
+      <nav>
+        <ul>
+          <li><a href="homepage.html">Home</a></li>
+          <li><a href="aboutme.html">About Me</a></li>
+          <li><a href="project.html">Projects</a></li>
+          <li style="float: right"><a href="contact.html">Contact</a></li>
+        </ul>
+      </nav>
+    </div>
+    <div class="heading">
+      <h1>my project</h1>
+    </div>
+    <div class="container">
+      <section class="about"></section>
+      <div class="row">
+        <div class="flex">
+          <div class="about-image"></div>
+          <div class="about-content">
+            <img src="image/websiteone.jpg" height="400" width="500" />
+            <h2>Unicorn gundam wiki webpages</h2>
+            <br />
+            <p>
+              for this project this was the frist time coding my own wiki that
+              would mostly show information and images of a gundamn name unicorn.
+              for this wiki i started with something simple with coding in the images size
+              and postion next adding the text about the unicorn and placing it under the image then 
+              under text is the made up table and lastly would be adding a background.
+              this project will act as a starter base for me learning the basic of html and class
+              as a begginner.
+            </p>
+          </div>
+        </div>
+        <div class="flex">
+          <div class="box">
+            <img src="image/Appcard.jpg" height="400" width="600" />
+            <h2>Trading card game guide App</h2>
+            <p>
+              For this project is mostly for my assisment which was to create a project that relate
```

Comment:
just wondering, did you ChatGPT or Google Gemini to correct grammer and spelling?